resources/meilisearch-release.md

```diff
@@ -148,8 +148,8 @@ It happens some releases come with impactful bugs in production (e.g. indexation
 git fetch origin --tags --force
 
 # Create the branch
-git checkout latest
-git checkout -b release-vX.Y.Z
+git checkout release-vX.Y.Z # The latest release you want to patch
```

Comment:
Why not using `latest` if you have `git fetch origin --tags --force` right before? It works too